Content.Client/Atmos/EntitySystems/AtmosDebugOverlaySystem.cs

```diff
@@ -22,7 +22,7 @@ internal sealed class AtmosDebugOverlaySystem : SharedAtmosDebugOverlaySystem
         /// <summary>Gas ID used by GasMoles mode</summary>
         public int CfgSpecificGas = 0;
         /// <summary>Uses black-to-white interpolation (as opposed to red-green-blue) for colourblind users</summary>
-        public bool CfgCBM = false;
+        public bool CfgCbm = false;
```

Comment:
Do not blindly follow IDE suggestions for renaming variables. "CBM" stands for "color-blind mode" which is an acronym. This should stay capitalized.

Reply:
Oh okay, will get it back